### DISCUSSION TITLE
would like to ask for help in UVTools exposure finder fail

### DISCUSSION THREAD

**Discussion by claymore1984** [sn4k3/UVtools#748] [Q&A]
Greetings,

 I'm an absolute noob to 3D printing read saw a lot of info but no real actual printing experience. I saw a YT video to find the optimal exposure for my printer resin with the least fails, but sadly even the test print failed 2/2 times.
 
 The printer is an anycubik photon mono x 6K (screen protector installed) the resin is Elegoo Water washable Grey. The printer has already been used by one of my friends for a simple print since then 3 things changed I got a WhamBam flex plate I used kapton tape on the edges if resin leaks noting should spill in to the machine and I use a different resin ( he used anycubic ones)

 The fail that I get is a part of the print sticks to the plate the other to the fep the ticker part was on the build plate on the first print it was smooth on bot side on both parts on this one I se a kind of layer separation on the ticker part wich was on the plate.

 Could you give me some pointers how I should troubleshoot this? Both times I re leveled the plate with the "leveling paper" anycubic provided and used the test print for UVtools to get the settings fro my printer the tests where once 23x4 base 1,75-3.00 0.25 incr. second 25x4 base 2.00-3.00 0.20 incr and I tested the printer and it should be able to do the test. (maybe screen protector film and kapton tape together is to thick and this causes the problem??? os the Flex plate?)

Thx
And best regards
Zoltan

![PXL_20230829_215425397 MP](https://github.com/sn4k3/UVtools/assets/143608373/49802b3b-75c9-4fc7-9726-77b83ecee450)
![PXL_20230829_215429918 MP](https://github.com/sn4k3/UVtools/assets/143608373/f065e677-d0a0-4e40-999c-1d96f4474714)
![PXL_20230829_215436430 MP](https://github.com/sn4k3/UVtools/assets/143608373/a189a513-6d0a-4bd8-976b-6c64cdb52723)
![PXL_20230829_215510472 MP](https://github.com/sn4k3/UVtools/assets/143608373/0de08ac7-0ffa-4950-b722-ab867ebf8bdd)
![PXL_20230829_221834883 MP](https://github.com/sn4k3/UVtools/assets/143608373/1e1d164f-43f5-4a35-bfd5-91a4679f671f)

**Comment by sn4k3** [comment]
The multiple exposure test is not compatible with Anycubic printers. Firmware restrictions prevent you from use same layers position which cause the problem.
